### PR TITLE
WC: Documentation page

### PR DESCRIPTION
**Pull request by Prabhav-Sharma**
Added Documentation pages for all components
- Added JS file for hamburger nav implementation

### CODE REVIEW

File: Docs/alert.html [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+                    </p>
+                    <h3 class="component-title">Basic Alerts</h3>
+                    <p class="para-m">
+                        These alerts consist of basic texts. To use these alerts, one must use the below code and can change the class for 
+                        the type of alert <span class="highlight-text">"primary-alert","success-alert","warning-alert", "error-alert"</span>
+                             for primary, success, warning and error alerts respectively, as the naming suggests.
+                    </p>
+                    <div class="code-example flex-row">
+                        <pre><code class="language-html">&lt;div class="alert primary-alert"&gt; 
+&lt;p class="alert-text"&gt;A simple
```
Comment (ArjunGTX) [inline Docs/alert.html:79]: Proper indentation would have been better

File: Docs/alert.html [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+                        </div>
+                    </div>
+                    <h3 class="component-title">Alerts with Icons</h3>
+                    <p class="para-m">
+                        For alerts with icons, the below code can be used, the aforementioned instructions are applicable, one need 
+                        only change the type of alert
+                    </p>
+                    <div class="code-example flex-row">
+                        <pre><code class="language-html">&lt;div class="alert-icon primary-alert"&gt;
+    &lt;img class="icon icon-alert" src="/CSS/Components/assets/info.png"&gt;
```
Comment (ArjunGTX) [inline Docs/alert.html:105]: Same as above

**Comment by ArjunGTX** [comment]
Also Depoly preview is not working. Not sure if its just for me. Everything else looks good

File: Docs/alert.html (hunk 1 above)
Comment (Prabhav-Sharma) [reply Docs/alert.html:79]: Yeah, I tried that but then the highlight js code example box takes up more space and isn't responsive. This is done on purpose

**Review by SukanyaSen530 — APPROVED** [review]
Home page image is breaking. Code looks fine.

**Pull request merged**